2023-02-22 05:27:36,131 [MainThread  ] [INFO ] [root]  STARTING BROWSER: /home/jake/research/chromium/src/out/direct_handles_release/chrome
2023-02-22 05:27:36,131 [MainThread  ] [INFO ] [root]  STARTING BROWSER: driver: /home/jake/research/chromium/src/out/direct_handles_release/chromedriver
2023-02-22 05:27:36,131 [MainThread  ] [INFO ] [root]  STARTING BROWSER: args: --no-default-browser-check --disable-component-update --disable-sync --no-experiments --enable-benchmarking --disable-extensions --no-first-run --disable-background-timer-throttling --disable-renderer-backgrounding --window-size=1500,1000 --enable-logging --log-file=/home/jake/research/v8_handles_experiments/crossbench/chrome_v112_chrome-direct-no-compact_js_flags_no_compact_with_stack/speedometer_2.1/4/browser.chrome.log --js-flags=--no-compact-with-stack
2023-02-22 05:27:39,437 [MainThread  ] [INFO ] [root]  RUNNING STORY
2023-02-22 05:29:29,102 [MainThread  ] [WARNI] [urllib3.connectionpool]  Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcaacdfb20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/305624cef264152f08fdf546e13a8531
2023-02-22 05:29:29,102 [MainThread  ] [WARNI] [urllib3.connectionpool]  Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcaa358be0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/305624cef264152f08fdf546e13a8531
2023-02-22 05:29:29,103 [MainThread  ] [WARNI] [urllib3.connectionpool]  Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcaa358790>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/305624cef264152f08fdf546e13a8531
